src/ResourceManager/KeyVault/Commands.KeyVault/help/Set-AzureKeyVaultCertificateAttribute.md

```diff
@@ -1,4 +1,4 @@
-﻿---
```

Comment:
@maddieclayton it looks like the underlying call for this cmdlet is a [`PATCH`](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault/Generated/KeyVaultClient.cs#L9769), so we should look at renaming this cmdlet to `Update-AzureKeyVaultCertificateAttribute` with an alias to `Set-*`

Reply:
Going to rename to Update-AzureKeyVaultSecret as I will combine this with Set-AzureKeyVaultSecret at the breaking change release and I don't want to aliases to get out of control.

Reply:
Please don't do this. There is no separate command for adding or importing a secret - Set fits the intended profile here.

Reply:
I'm going to skip doing these aliases for now and we will discuss again for the breaking change PR - I think some of these aliases might make sense while others do not.